src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -360,8 +348,16 @@
       <Output TaskParameter="ConsoleOutput" PropertyName="VersionHostName" />
     </Exec>
 
-    <PropertyGroup>
-      <BuiltByString Condition="'$(BuiltByString)' == '' AND '$(VersionUserName)' != '' AND '$(VersionHostName)' != ''">%20built by: $(VersionUserName)-$(VersionHostName)</BuiltByString>
+    <!-- Set BuiltByString (which is added to the end of the informational version) -->
+    <!-- This string has additional information as well (like the GIT commit URL) -->
+    <!-- and has the general form @Name1: Value1 @Name2: Value2  ... which makes it easy to parse -->
+    <PropertyGroup Condition="'$(BuiltByString)' == ''">
+      <!-- Set the @BuiltBy key-value pair -->
+      <BuiltByString Condition="'$(VersionUserName)' != '' AND '$(VersionHostName)' != ''">$(BuiltByString)%20%40BuiltBy: $(VersionUserName)-$(VersionHostName)</BuiltByString>
```

Comment:
Ah, I didn't notice it was already there. It seems like the original thought was that MSBuild removes leading non-encoded spaces, but I've never seen that happening. I think changing the `%20` to ` ` now would be good so this is more readable in the future.

The `%40` for `@` makes sense, thanks.